sdk/formrecognizer/Azure.AI.FormRecognizer/README.md

```diff
@@ -1,136 +1,339 @@
 # Azure Cognitive Services Form Recognizer client library for .NET
-Azure Cognitive Services Form Recognizer is a cloud service that uses machine learning to extract text and table data from form documents. It allows you to train custom models using your own forms, to extract field names and values, and table data from them.  It also provides a prebuilt models you can use to extract values from receipts, or tables from any form.
+Azure Cognitive Services Form Recognizer is a cloud service that uses machine learning to recognize form fields, text, and tables in form documents.  It includes the following capabilities:
+
+- Recognize Custom Forms - Recognize and extract form fields and other content from your custom forms, using models you train with your own form types.
+- Recognize Form Content - Recognize and extract tables, lines and words in forms documents, without the need to train a model.
+- Recognize Receipts - Recognize and extract common fields from US receipts, using a pre-trained receipt model.
 
 [Source code][formreco_client_src] | <!--[Package (NuGet)]() | [API reference documentation]() |--> [Product documentation][formreco_docs] <!--| [Samples]()-->
```

Comment:
we should enable the other links too.
- For nuget, we can guess the end result link: `https://www.nuget.org/packages/Azure.AI.FormRecognizer` 
- API ref we can add an aka.ms link
- Samples: link will be broken for now, but once u merge the PR it should work